<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1461</width>
    <height>971</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QQ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qqicons.qrc">
    <normaloff>:/qqicons/qrc/QQicon.svg</normaloff>:/qqicons/qrc/QQicon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_11" rowstretch="1,1000,1" columnstretch="1,1000,1">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <spacer name="verticalSpacer_21">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>7</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_35">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>12</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="majorSectionHLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="LeftBar" name="frmTab">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stkwThumbnail">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>441</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>441</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageInfoList">
         <layout class="QGridLayout" name="gridLayout_7" rowstretch="1,1000,1" columnstretch="1,1000,1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <spacer name="verticalSpacer_12">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>6</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_18">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_20">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>48</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_13">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>98</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QScrollArea" name="sldInfoList">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="InfoListWidget" name="frmInfoList">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>437</width>
               <height>961</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageFriends">
         <layout class="QGridLayout" name="gridLayout_9" rowstretch="1,1000,1" columnstretch="1,1000,1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <spacer name="verticalSpacer_17">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>6</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_24">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="10,55">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="wFriendsFunc" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>148</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>148</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget{
	border-top: 1px solid black;
	border-bottom: 1px solid black;
	border-left: 1px solid black;
	border-right:1px solid black;
	background-color: rgb(255, 255, 255);
}</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_8" rowstretch="8,100,1" columnstretch="1,1000,1">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="1">
                <spacer name="verticalSpacer_15">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>6</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="0">
                <spacer name="horizontalSpacer_22">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>1</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="1">
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <property name="spacing">
                  <number>8</number>
                 </property>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,6,1">
                   <item>
                    <spacer name="horizontalSpacer_19">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>38</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btnAddFriend">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>59</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>59</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <family>微软雅黑</family>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QPushButton{
	color: rgb(35, 35, 35);
	background-color: rgb(255, 255, 255);
	border: 1px solid;
	border-color: rgb(211, 211, 211);
	border-radius: 10px;
}

QPushButton:hover{
	background-color: rgb(236, 236, 236);
}

QPushButton:pressed{
	color: rgb(108, 108, 108);
	background-color: rgb(224, 224, 224);
}</string>
                     </property>
                     <property name="text">
                      <string>添加好友</string>
                     </property>
                     <property name="icon">
                      <iconset resource="../qqicons.qrc">
                       <normaloff>:/qqicons/qrc/add_friend.svg</normaloff>:/qqicons/qrc/add_friend.svg</iconset>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_21">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>38</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="ExpandButton" name="btnFriendsInfo">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>61</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>61</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton{
	text-align: left;
	border-top: none;
	border-bottom: none;
	padding: 0px 20px 0px 20px;
	color: rgb(42, 42, 42);
	background-color: rgb(255, 255, 255);
	/*border-radius: 5px;*/
}

QPushButton:hover{
	
	background-color: rgb(240, 240, 240);
}

QPushButton:pressed{
	background-color: rgb(224, 224, 224);
}</string>
                   </property>
                   <property name="text">
                    <string>好友通知</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="2">
                <spacer name="horizontalSpacer_23">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>38</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="1">
                <spacer name="verticalSpacer_14">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>3</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="FriendListWidget" name="trFriendLists">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="indentation">
               <number>0</number>
              </property>
              <property name="columnCount">
               <number>2</number>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>count</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_26">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>68</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_16">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>58</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stkwDetail">
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageNull">
         <property name="styleSheet">
          <string notr="true">QWidget{
	background-color: rgb(242, 242, 242);
	background-image: url(:/qqicons/qrc/qq_pure.svg);
	background-repeat: no-repeat;
	background-position: center;
}</string>
         </property>
        </widget>
        <widget class="ChatMessageWidget" name="pageChatMessage"/>
        <widget class="FriendInfoWidget" name="pageFriendInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
        <widget class="FriendNoticeWidget" name="pageFriendNotice"/>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_36">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>118</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_22">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>68</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ExpandButton</class>
   <extends>QPushButton</extends>
   <header location="global">FriendListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>FriendListWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">FriendListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>LeftBar</class>
   <extends>QFrame</extends>
   <header location="global">LeftBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatMessageWidget</class>
   <extends>QWidget</extends>
   <header location="global">ChatMessageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FriendInfoWidget</class>
   <extends>QWidget</extends>
   <header location="global">FriendInfoWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FriendNoticeWidget</class>
   <extends>QWidget</extends>
   <header location="global">FriendNoticeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InfoListWidget</class>
   <extends>QWidget</extends>
   <header location="global">InfoListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qqicons.qrc"/>
  <include location="../qqicons.qrc"/>
 </resources>
 <connections/>
</ui>
